### DISCUSSION TITLE
The "Fork" function (button) should be confirmable

### DISCUSSION THREAD

**Discussion by NetLightDev** [community/community#14349] [Other Feature Feedback, Questions, & Ideas]
I think that the "Fork" function (button) that is available on each project's page on GitHub.com should require some confirmation, because this is not an usual function people use and it may be accidentally pressed, as it happend to me ;)
Now, the fork is created immediately, which may be completely unintended and accidental.
Also I might not want the project's owner to know that I tried to create a fork from his project...

**Comment by unitycoder** [comment]
please no, now it takes extra effort to make forks. (or at least make it optional to have that new fork creation window)
https://github.com/github/feedback/discussions/14745#discussioncomment-2551552
